src/NuGet.Status/App_Start/Startup.cs

```diff
@@ -55,15 +54,21 @@ public void Configuration(IAppBuilder app)
                     {
                         ClientId = _clientId,
                         Authority = _authority,
-                        RedirectUri = _redirectUri,
+                        RedirectUri = _rootUri + "/signin-oidc",
                         PostLogoutRedirectUri = _rootUri,
-                        TokenValidationParameters = new System.IdentityModel.Tokens.TokenValidationParameters
+                        CookieManager = new SystemWebChunkingCookieManager(),
+                        TokenValidationParameters = new Microsoft.IdentityModel.Tokens.TokenValidationParameters
                         {
                             ValidateIssuer = false,
                             RoleClaimType = "roles"
                         },
                         Notifications = new OpenIdConnectAuthenticationNotifications
                         {
+                            RedirectToIdentityProvider = notification =>
+                            {
+                                notification.ProtocolMessage.Prompt = "select_account"; // force entering of credential
+                                return Task.FromResult(0);
```

Comment:
`Task.CompletedTask`?

Reply:
Just following convention from below callback, but will change it.

Reply:
I was wondering about this. It may be worth leaving as-is--every example (as well as the below callback) seems to use `FromResult(0)`: e.g. https://stackoverflow.com/questions/31328378/changing-returnurl-in-owin-redirecttoidentityprovider-notification
...even though the return type is `Task` and not `Task<int>`. Potentially there is some secret sauce afoot here? At any rate, this seems to be the convention. We even do it in gallery code: https://github.com/NuGet/NuGetGallery/blob/master/src/NuGetGallery.Services/Authentication/Providers/AzureActiveDirectoryV2/AzureActiveDirectoryV2Authenticator.cs#L266
...but I can change it if you want. Just doesn't seem the done thing for some reason.

Reply:
No, it's just that these libraries pre-date Task.CompletedTask and the samples were never updated.

Reply:
Thanks, @Tratcher!